### DISCUSSION TITLE
add searx engine

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7979] [General] [answered]
Hi!

I did try to put searx.tiekoetter.com for replace duckngo engine but doesn't work. I do not know ho to do it.
I use Qutebrowser 3.0.

Thank you,.

**Comment by emanuele6** [comment, marked as answer]
You need to set the "DEFAULT" key of the `url.searchengines` setting to a search URL of that search engine, with the search term replaced by `{}`.
For `searx.tiekoetter.com`, you would do `https://searx.tiekoetter.com/?q={}`.
To set it, you just need to run:
```
:config-dict-add -r url.searchengines DEFAULT https://searx.tiekoetter.com/?q={}
```

Or, if you want to configure qutebrowser with python, you can do something like:
```py
c.url.searchengines = { 'DEAFULT': 'https://searx.tiekoetter.com/?q={}' }
```

**Reply by ghost** [reply]
It didn't work because I use set.

Thank you very much.
